packages/backend-data/API.md

```diff
@@ -56,6 +53,10 @@ export type OIDCAuthorizationModeProps = {
     tokenExpireFromIssueInSeconds: number;
 };
 
+// Warnings were encountered during analysis:
+//
+// src/types.ts:91:3 - (ae-forgotten-export) The symbol "DefaultAuthorizationMode" needs to be exported by the entry point index.d.ts

```

Comment:
check will complain about this, you need to export type.

Reply:
Yeah. Just fixed this.